src/main/reactjs/src/configs/i18n.ts

```diff
@@ -50,6 +56,7 @@ export const initI18n = () => {
   return use(initReactI18next)
     .use(LanguageDetector)
     .init({
+      fallbackNS: 'langs',
```

Comment:
"Besides defining multiple namespaces to load from, you also can set fallback namespaces. Thus, if a key to translate isn't found in the given namespace, it will look it up in the indicated fallbacks."

langs ei oikein pidä ehkä tässä paikansa, tarvitaanko tätä ollenkaan?

Reply:
Tuossa itse asiassa fallBackns toimii jotenkin oudosti. Sen voisi jättää vielä tällä tavalla.